packages/redis-7.2/packaging

```diff
@@ -4,7 +4,7 @@ set -e
 target_dir="redis_source"
 mkdir -p ${target_dir}
 
-tar xvf redis/redis-6.2.*.tar.gz -C ${target_dir} --strip-components 1
```

Comment:
Just a question, do we have to replace existing 6.x version with 7.x or we have to create a new package with 7.x